src/main/java/domain/TodoRepository.java

```diff
@@ -20,4 +20,10 @@ public long addTodo(final Todo todo){
         em.persist(todo);
         return todo.getId();
     }
+
+    @Transactional
+    public void deleteTodo(final long todoId) {
+        Todo todo = em.find(Todo.class, todoId);
```

Comment:
Was ist, wenn das To-Do nicht exitiert?

Reply:
dann sagt mir mein LOG "Delete todo by id: 657 not possible" und mein Server antwortet mit 404

Reply:
Das Exception handling geschieht doch bereits in der TodoResource ? !

Reply:
Ach, habe das ExceptionHandling übersehen.